app/services/sitemap_indexer.rb

```diff
@@ -2,14 +2,12 @@
 
 class SitemapIndexer
   attr_reader :domain,
-              :scheme,
               :searchgov_domain,
-              :uri
+              :sitemap_url
```

Comment:
I renamed `uri` to `sitemap_url`, as we no longer convert the `sitemap_url` string to a `URI`.